scripts/library/README.md

```diff
@@ -0,0 +1,30 @@
+# ukis-libraries-scripts
+
+This package contains a few scripts to build and test angular libraries.
+
+It is based on [depcheck](https://github.com/depcheck/depcheck) and [toposort](https://github.com/marcelklehr/toposort#readme)
+
+It uses the package scope `@dlr-eoc`, `main package.json` and the angular config file `angular.json`.
+
+
+- To get the list of all projects is reads the the angular config file (angular.json).
+- To check all dependencies it uses the package scope `@dlr-eoc` and `depcheck`.
+- To get the build order it used `depcheck` and `toposort`.
```

Comment:
To get the build order it uses `depcheck` and `toposort`.

Reply:
done https://github.com/dlr-eoc/ukis-frontend-libraries/pull/90/commits/57fd125197d825c2235226947b76964d69546ba2.